src/extension.ts

```diff
@@ -228,6 +282,10 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
   );
 }
 
+function updateJavaConfig(javaHome: string, global: boolean = true) {
```

Comment:
instead of a boolean, I would prefer this to use `ConfigurationTarget` instead of `boolean`.